### PR TITLE
Main readme update

### PR DESCRIPTION
**Pull request by akiskips**
The reference hyperlinks in the main features part are empty at the moment. After the review they will be updated.
Please feel free to comment / update

### CODE REVIEW

**Comment by akiskips** [comment]
> > The reference hyperlinks in the main features part are empty at the moment. After the review they will be updated.
> 
> imo this sentence alone prevents approval from my side.

Ok, I see your point. I added them. Feel free to add more (e.g. was not sure where it would be best the the deployment with docker and k8s should be pointing)

File: README.md [hunk 1]
```diff
@@ -61,20 +87,18 @@ Optionally, you can download and install [Postman](https://www.getpostman.com) t
...
-- Pull and run prebuilt images from our [Docker Hub repository](https://hub.docker.com/orgs/sapi40/repositories). Pulling can last several minutes depending on your internet connection.
+Setup locally:
 ```bash
-## 'up' (start) the composed setup
+## Option 1
+## Use images from Docker Hub (https://hub.docker.com/orgs/sapi40/repositories)
 $ make install
-```
 
-- Build the __i40-aas__ images from source first, then pull external dependencies and continue to run everything. Builds are executed in parallel, thus build logs can appear cluttered. Building all __i40-aas__ images can last several minutes.
```
Comment (Manu3756) [inline README.md]: why delete this description?

File: README.md [hunk 2]
```diff
@@ -61,20 +87,18 @@ Optionally, you can download and install [Postman](https://www.getpostman.com) t
 ### Getting Started
 Service interactions can be deployed, tested and developed locally using Docker Compose as described below. For further instructions & information about __i40-aas__ check [the documentation](docs/README.md)
 
-Run __i40-aas__ locally using any of these options:
```
Comment (Manu3756) [inline README.md]: looks like a wrong local merge. This description should not be deleted, as it is the new one.

File: README.md [hunk 3]
```diff
@@ -9,21 +9,47 @@ __i40-aas__ welcomes [contributions](#contributing). Please read about [known is
...
+
+## Main Features
+The AAS-Service provides an implementation of the RAMI 4.0 reference  architecture as specified by [Plattform Industrie 4.0](https://www.zvei.org/themen/industrie-40/details-of-the-asset-administration-shell/). Main features include:
+
+- Standardized and secure communication interfaces and API to access Asset Administration Shells of the digital representation of a physical asset (Digital Twin), that enable:
+  - use of a single AAS information model
+  - secure access to asset information from cloud and edge
+  - integration of assets without a communication interface (i.e. passive), e.g. via barcodes or QR-codes pointing to asset       management solutions
+
+- Facilitate Interoperability:
```
Comment (Manu3756) [inline README.md]: Do not use links if there is no linked page.

**Pull request closed without merging**